### DISCUSSION TITLE
关于XHS-Spider的需求征集

### DISCUSSION THREAD

**Discussion by xisuo67** [xisuo67/XHS-Spider#9] [Q&A] [answered]
如题

**Comment by xisuo67** [comment, marked as answer]
- 后期规划：
  - 用户笔记详情
    - 用户所有笔记中增加查看详细笔记功能
    - 搜索框支持笔记详情链接，可直接预览详细笔记
  - 可视化下载列表
    - 分为已完成、未完成两项下载列表
    - 所有下载任务可见
    - 暂停开始任务
    - 删除，添加任务
   - 程序启动自动cookie填充
   - 搜索支持搜索关键字
   - 数据持久化
     - 保存用户个性化设置

**Reply by W2426566602** [reply]
666 我来发一个

**Comment by W2426566602** [comment]
页面能不能做成动态效果嘤嘤嘤
